File: api/Service/UserService.cs
using System.Linq;
using Microsoft.EntityFrameworkCore;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Mvc;

namespace api.Service
{
  public class UserService : BaseService<Models.User>, Interface.IUserService
  {
    public UserService(Database.DBContext context) : base(context)
    {
      this._Table = this._context.Users;
    }

    [HttpGet]
    public async Task<Models.User> GetByLoginPswrd(string login, string pswrd)
    {
      return await this._Table.Where(x => x.Login == login && x.Pswrd == pswrd).SingleOrDefaultAsync();
    }
  }
}

File: app/src/service/user.service.ts
import { Injectable } from '@angular/core';
import { BaseService } from './base.service';
import { User } from 'src/models/user';
import { UserRepository } from 'src/repository/user.repository';
import { Observable } from 'rxjs';

@Injectable()
export class UserService extends BaseService<User>
{
  constructor(protected _repository: UserRepository)
  {
    super(_repository);
  }

  public getByLoginPswrd(login: string, pswrd: string) : Observable<User>
  {
    return this._repository.getByLoginPswrd(login, pswrd);
  }
}
